cdap-ui/app/cdap/components/Experiments/DetailedView/ModelsTable/index.js

```diff
@@ -118,153 +100,166 @@ const getNewHeadersBasedOnOutcome = (outcomeType) => (
     addMetricsToHeaders(tableHeaders, categoricalMetrics)
 );
 
-const renderTableHeaders = (outcomeType, renderSortableTableHeader) => {
+const addDetailedModelObject = (list) => {
+  let activeIndex = list.findIndex(model => model.active);
+  if (activeIndex !== -1) {
+    return [
+      ...list.slice(0, activeIndex + 1),
```

Comment:
Just to be sure, are you adding something after `activeIndex`, or are you replacing the value at `activeIndex` here?

Reply:
After.